### DISCUSSION TITLE
[Feature] local url

### DISCUSSION THREAD

**Discussion by KrzysztofMoskalik** [immich-app/immich#9949] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Let's introduce an local url functionality. If you are connected to specific wifi ssid, app will use an second, local url instead of public one.
This functionality works perfectly in Home Assistant app. This will speed up content synchronization with home serwer when you are at home and still allow to use public url. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#308
